src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -200,7 +197,9 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdThe
       if (animateTo === animateFrom) {
         path.attr('d', getSvgArc(animateTo, diameter, pathDiameter, rotation));
       } else {
-        lastDrawFrame = $$rAF(function animation(now) {
+        // Note that this shouldn't use use $$rAF, because it can cause an infinite loop
+        // in any tests that call $animate.flush.
+        $window.requestAnimationFrame(function animation(now) {
```

Comment:
We do lose all that, however we don't support any browsers that either don't have it, or use the prefixed version.